### PR TITLE
Support experimental packages

### PR DESCRIPTION
**Pull request by conorsch**
First pass at adding support for the "test" branch of grsecurity patches, as described in #10. The implementation proposed in this PR works reasonably well, but does not prompt for vars authentication information for restricted "stable" downloads. In the interest of keeping the diffs readable for review, I'm submitting now, targeting `develop`, and will add variable prompts after further testing.

### CODE REVIEW

File: ansible/roles/build-grsec-kernel/tasks/fetch_files.yml [hunk 1]
```diff
@@ -1,50 +1,24 @@
...
-  args:
-    creates: "{{download_directory}}/ubuntu-package"
-  when: target_os == 'ubuntu'
 
-- name: copy ubuntu overlay control files
-  shell: >
-     cp {{download_directory}}/ubuntu-trusty/debian/control-scripts/p* \
-        {{download_directory}}/ubuntu-package/pkg/image/
-  when: target_os == 'ubuntu'
+- name: Fetch grsecurity patch and signature.
```
Comment (ageis) [inline ansible/roles/build-grsec-kernel/tasks/fetch_files.yml]: Can you explain why this is a nop?

**Comment by ageis** [comment]
The changes in this PR look good to me. But for the test patch, don't you need to include downloads for the linux-4.1.7 kernel?

File: ansible/roles/build-grsec-kernel/tasks/fetch_files.yml (hunk 1 above)
Comment (conorsch) [inline ansible/roles/build-grsec-kernel/tasks/fetch_files.yml]: Placeholder during development! The actual logic has moved to `tasks/fetch_grsecurity_files.yml`. Cleaned up the task lists in 989b7be. Thanks for catching!

**Comment by conorsch** [comment]
@ageis You're right, the problem with the experimental patches was that the Linux kernel version wasn't being set appropriately. I've added logic to associate the "longterm" kernel version with the "stable" grsecurity patches, and the "stable" (`4.x`) kernel version with the "test" grsecurity patches.

Choosing kernel versions dynamically makes it difficult to hardcode the checksums for downloaded files, so I'll add additional tasks to handle fetching the remote file in future commits (perhaps better suited to a separate PR).

**Comment by ageis** [comment]
@conorsch Excellent! With those two issues resolved, looks good to go now.

**Comment by conorsch** [comment]
@ageis :metal:

**Pull request merged**